src/react/account/AccountPoliciesList.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Box, Button } from '@scality/core-ui/dist/next';
+import { spacing } from '@scality/core-ui/dist/style/theme';
+import { formatShortDate } from '../utils';
+import { useIAMClient } from '../IAMProvider';
+import CopyARNButton from '../ui-elements/CopyARNButton';
+import { Tooltip } from '@scality/core-ui';
+import { SpacedBox } from '@scality/core-ui/dist/components/spacedbox/SpacedBox';
+import { notFalsyTypeGuard } from '../../types/typeGuards';
+import { useMutation, useQuery } from 'react-query';
+import { queryClient } from '../App';
+import DeleteConfirmation from '../ui-elements/DeleteConfirmation';
+import { Banner } from '@scality/core-ui';
+import { getPoliciesQuery } from '../queries';
+import { Icon } from '../ui-elements/Help';
+import AwsPaginatedResourceTable from './AwsPaginatedResourceTable';
+import IAMClient from '../../js/IAMClient';
+
+const STORAGE_ACCOUNT_OWNER_POLICY = 'storage-account-owner-policy';
+const STORAGE_MANAGER_POLICY = 'storage-manager-policy';
+
+const RenderEditButton = ({
+  policyName,
+}: {
+  policyName: string;
+  path: string;
+}) => {
+  const history = useHistory();
+  const isEditPolicyDisabled = [
+    STORAGE_MANAGER_POLICY,
+    STORAGE_ACCOUNT_OWNER_POLICY,
+  ].includes(policyName);
+  return (
+    <SpacedBox ml={12}>
+      <Button
+        style={{ height: spacing.sp24 }}
+        disabled={isEditPolicyDisabled}
+        variant="secondary"
+        label="Edit"
+        icon={<i className="fa fa-pen"></i>}
+        onClick={() => history.push(`users/${policyName}/update-policy`)}
+        tooltip={{
+          overlayStyle: {
+            width: '16.5rem',
+          },
+          overlay: isEditPolicyDisabled
+            ? 'You cannot edit this predefined Scality Policy'
+            : '',
+        }}
+      />
+    </SpacedBox>
+  );
+};
+
+const RenderAttachButton = () => {
+  const history = useHistory();
+  return (
+    <SpacedBox ml={12}>
+      <Button
+        style={{ height: spacing.sp24 }}
+        variant="secondary"
+        label="Attach"
+        icon={<i className="fas fa-link"></i>}
+        onClick={() => history.push('attach-user-policy')}
```

Comment:
I'm not convinced about this path. Likely we could change it to:
```suggestion
        onClick={() => history.push(`/policies/${policyName}/attachments`)}
```